Shared/CommonAppDelegate.swift

```diff
@@ -137,7 +138,8 @@ open class CommonAppDelegate: ParticlesAppDelegate {
 
     open func injectRating() {
         Console.shared.log("injectRating")
-        RatingService.shared = AppStoreRating(threshold: 20)
+        //TODO: Comment in when rate pre-prompt UI is ready
+//        RatingService.shared = dydxPointsRating()
```

Comment:
note this service is turned off for now until designs are finalized

Reply:
Maybe adding a feature flag?